logrotate.8.in

```diff
@@ -606,57 +608,68 @@ the second argument to the \fBpostrotate\fR script.  If the scripts exit with
 error, the remaining actions will not be executed for the affected log only.
 
 .TP
-\fBfirstaction\fR/\fBendscript\fR
-The lines between \fBfirstaction\fR and \fBendscript\fR (both of which
-must appear on lines by themselves) are executed (using \fB/bin/sh\fR) once
+\fBfirstaction\fR/\fIscript\fR/\fBendscript\fR
```

Comment:
How did `/script/` appear in all these headings?  As far as I know, there is no `script` configuration directive.  I think it should really say `firstaction/endscript`, `postrotate/endscript` and the like.  Am I missing anything?

Reply:
Thanks for the prompt review.  

It seemed unclear that the script (the whole point of the directive) wasn't shown in the heading.

My reasoning: The same way (for example) the rotate directive takes an argument of count, the various *script directives take an argument of the script.  So I added it in italic font just as count is.

Reply:
OK, got it.  Then I would probably also replace the slashes by plain spaces to make it look like the other configuration directives that take an argument.  Would that work for you?

Reply:
Thanks for pushing on this - I had considered it and given up.  Space is a poor choice since in other places space is a space and in this case it's a syntax error.  But what do you think of this:

**firstaction**
_script_
**endscript**

The source for this is:
.TQ
\fBfirstaction\fR
.TQ
\fIscript\fR
.TQ
\fBendscript\fR

Reply:
Yes, using space yields the same syntax error as using slash.  Putting there newlines could be an option.  In that case, I would also indent `script` as it is done in the examples on top of the `CONFIGURATION FILE` section.

Reply:
Good idea.  I'll look into that.  BTW the .TQ macro is specific to groff.  Is that an issue?

Reply:
To answer my own question - yes, it's an issue.  MacOS ships with groff but not the .TQ macro, so I'll include it in the file.  I have a working indent.  I should have an update in a couple hours.

Reply:
Looks good.  Thanks for the update!